### PR TITLE
Ali

### PR DESCRIPTION
**Pull request by alimeysam**
The deleteFile.js has been implemented to delete file from database. Licence and README files for highlight.js have been added to the public/libs/highlightjs directory.

### CODE REVIEW

**Review by AdamCollins — CHANGES_REQUESTED** [review]
You seem to be missing some crucial files (namely the libs/bootstrap and jquery). Could you please pull from dev to your branch, resolve any merge conflicts in your branch and commit/push again so that these files are not removed?

**Comment by alimeysam** [comment]
I have pulled from dev, solved merge conflicts and pushed back to My branch, now my branch is also up to date

File: routes/deleteFile.js [hunk 1]
```diff
@@ -1,3 +1,32 @@
...
+    //Search for the page
+
+    Page.deleteOne({'page_id':pageID}, function (err, rst) {
+
+        //if result found
+        if(rst) {
+            res.sendStatus(200);
+
+        } else {
+            fs.unlink('./doc/' + pageId + '.txt');
```
Comment (AdamCollins) [inline routes/deleteFile.js]: What are you unlinking in the file system?

File: public/libs/highlightjs/README.md [hunk 2]
```diff
@@ -0,0 +1,150 @@
+# Highlight.js
```
Comment (AdamCollins) [inline public/libs/highlightjs/README.md]: We don't need to include the README file.

File: public/libs/highlightjs/README.ru.md [hunk 3]
```diff
@@ -0,0 +1,142 @@
+# Highlight.js
```
Comment (AdamCollins) [inline public/libs/highlightjs/README.ru.md]: Нам не нужно включать файл readme. Ты не украинский, не русский

File: views/home.handlebars [hunk 4]
```diff
@@ -0,0 +1,12 @@
+<!doctype html>
```
Comment (AdamCollins) [inline views/home.handlebars:1]: Please delete this file. We dont need it anymore

**Review by AdamCollins — CHANGES_REQUESTED** [review]

**Pull request merged**